composer.json

```diff
@@ -163,9 +163,7 @@
     "silex/web-profiler": "~1.0.8",
     "symfony/web-profiler-bundle": "*",
     "symfony/var-dumper": "*",
-    "site5/phantoman": "^1.1",
```

Comment:
I removed because we are not using it. This installs phantomJS to the image. But since we are using alpine linux, it doesn't work. So I think it's useless having it there. But I can change it back and leave it there. It doesn't really matter